src/rgw/rgw_cr_tools.cc

```diff
@@ -283,7 +283,8 @@ int RGWBucketGetSyncPolicyHandlerCR::Request::_send_request()
   int r = store->ctl()->bucket->get_sync_policy_handler(params.zone,
                                                         params.bucket,
                                                         &result->policy_handler,
-                                                        null_yield);
+                                                        null_yield,
+                                                        dpp);
   if (r < 0) {
     lderr(cct) << "ERROR: " << __func__ << "(): get_sync_policy_handler() returned " << r << dendl;
```

Comment:
@cbodley can/should lderr be changed to use a `dpp` the way ldout can?

Reply:
@TRYTOBE8TME take note above ^

Reply:
Okay!